### DISCUSSION TITLE
Where should we put typescript fixes?

### DISCUSSION THREAD

**Discussion by saiichihashimoto** [sanity-io/sanity#3352] [Sanity Studio Support] [answered]
Hi, there's some incorrect or missing types in @sanity/types. Is it worth making PRs against it or is there somewhere new we should be doing this for v3?

**Comment by kmelve** [comment, marked as answer]
@saiichihashimoto It depends where the incorrect/missing types are! 

- If they are in v3 (`dev-preview`) then the https://github.com/sanity-io/sanity/tree/v3 branch.
- If they are in v2, then the https://github.com/sanity-io/sanity/tree/next branch.

Appreciate it!

**Reply by saiichihashimoto** [reply]
@kmelve I'm currently relying on the types from `2.29.5-purple-unicorn.856`. Which branch do those live in?

**Reply by saiichihashimoto** [reply]
I'm assuming those are the v3 types?

**Reply by kmelve** [reply]
That would be the `v3` branch (btw, you should probably also upgrade to the latest version on the `dev-preview` tag).
